Gordon360/Models/ViewModels/MembershipViewModel.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Gordon360.Models.CCT;
+using System;
+namespace Gordon360.Models.ViewModels
+{
+    public class MembershipViewModel
```

Comment:
Is this view model actually being used anywhere? I thought you had basically replaced all references to it with the MembershipView?